src/main/java/com/coveros/selenified/element/check/Contains.java

```diff
@@ -371,21 +371,21 @@ default int checkColumns(int numOfColumns, double waitFor, double timeTook) {
      * @param timeTook  - the amount of time it took for wait for something (assuming we had to wait)
      * @return Integer: the number of columns of the element. -1 will be returned if the element isn't present or a table
      */
-    default int checkRows(int numOfRows, double waitFor, double timeTook) {
+    int checkRows(int numOfRows, double waitFor, double timeTook) {
         // record the action, and check for table
-        if (!isPresentTable(getElement().prettyOutput() + " with the number of table rows equal to <b>" + numOfRows + "</b>", waitFor)) {
+        if (!isPresentTable(this.element.prettyOutput() + " with the number of table rows equal to <b>" + numOfRows + "</b>", waitFor)) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal " with the number of table rows equal to <b>" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1192)